### PR TITLE
Bootstrap 3 migration

### LINKED ISSUE #2: Mobile: broken menu button
Can be reproduced in responsive mode.
Can be compared with https://getbootstrap.com/docs/3.4/examples/navbar/
Can be related to bootstrap update...

### PR DESCRIPTION
**Pull request by RblSb**
Check buttons please.

Also, where to parse `program/$uid/run?r=123&theme=dark` request to have dark mode in embed output?
Upd: probably not needed, can be done with iframe attr.

Closes #2

### CODE REVIEW

**Comment by AlexHaxe** [comment]
it looks pretty good! 
I haven't tested it on an actual mobile device, but Chrome's simulator shows it working. I'll have to rebuild my local docker setup so it allows access outside of localhost to test it.

`Run standalone` button seems to miss a style in light mode and isn't aligned on right side (not sure if that was a deliberate change).

having Dark/Light mode toggle in embedded view is a nice addition, but I guess most people embedding snippets want that snippet window to match their site's style. so they select light or dark mode via `iframe` tag when embedding and usually have no interest in giving users control over that setting.

**Comment by RblSb** [comment]
I can remove that toggle if you want. Also, seems like you cannot get `frameElement` (and dark-theme attribute) from different domain, so i will move this to url parameter.

File: assets/templates.html [hunk 1]
```diff
@@ -135,12 +135,9 @@
...
     $$("#code2").style.display = "none";
     $$("#results").style.display = "block";
-    if (document.body.classList.contains("dark-theme")) {
-      $$("#iframe").setAttribute("dark-theme", "true");
-    } else {
-      $$("#iframe").removeAttribute("dark-theme");
-    }
-    $$("#iframe").setAttribute("src", "$frameUrl" + Math.random());
+    var url = "$frameUrl" + Math.random();
+    if (document.body.classList.contains("dark-theme")) url += "?theme=dark";
```
Comment (AlexHaxe) [inline assets/templates.html:139]: should be `"&theme=dark"`

**Comment by AlexHaxe** [comment]
thanks for your work on this, it's online now!

**Pull request merged**